README.md

```diff
@@ -1,85 +1,135 @@
+for the [Open Build Service](http://openbuildservice.org/) which all
+assist with packaging source code from SCM (source code management)
+repositories into tarballs.  The authoritative source is
+https://github.com/openSUSE/obs-service-tar_scm.
+
+## Services
+
+### tar_scm *(deprecated)*
+
+`tar_scm` is the legacy source service used to create a source tarball
+from one of the supported SCM (source code management) tools: `git`,
+`hg`, `svn`, and `bzr`.
 
-It supports many options, e.g. it can adjust resulting tarball
-parameters, include or exclude particular files when creating this
-tarball or generate an rpm changelog from the SCM commit log. For the
-full list of options please see tar_scm.service.in.
+`tar_scm` supports many options, e.g. it can adjust resulting tarball
+parameters, include or exclude particular files when creating the
+tarball, or generate an `rpm` changelog from the SCM commit log. For the
+full list of options please see `tar_scm.service.in`.
 
-Apart from various SCM like git, hg, bzr or svn it additionally
+Apart from various SCM like git, hg, bzr or svn, it additionally
 supports `--url` option that allows you to specify URL of the upstream
 tarball to be downloaded.
 
-This is the git repository for
-[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm).
-The authoritative source is
-https://github.com/openSUSE/obs-service-tar_scm
-
-The service can be used in combination with other services like
+`tar_scm` can be used in combination with other services like
 [download_files](https://github.com/openSUSE/obs-service-download_files),
 [extract_file](https://github.com/openSUSE/obs-service-extract_file),
 [recompress](https://github.com/openSUSE/obs-service-recompress) or
 [set_version](https://github.com/openSUSE/obs-service-set_version)
 e.g. within the [GIT integration](https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#Example_2:_GIT_integration)
 workflow.
 
-## Archive Formats
+**`tar_scm` is deprecated in favour of `obs_scm`.**
 
-### obscpio
+### obs_scm
 
-The `obscpio` archive format is the new OBS specific archive format
-based on `cpio` which can be consumed and generated by the [OBS Delta
-Store](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.architecture.html#delta_store)
-to store changes in an incremental way independently from your chosen
-SCM.  The `obs_scm` service generates archives in this format, which
-is basically the `cpio` `newc` format.
+`obs_scm` is similar in concept to `tar_scm`, but instead of directly
+generating tarballs, it instead uses the new `obscpio` archive format
+(see below) as an intermediate space-efficient format in which to
+store the sources.
+
+**It is recommended to use `obs_scm` in favour to `tar_scm`**, because
+it helps to save a lot of disk space on the server side, especially
+when used in continuous integration (e.g. nightly builds and builds of
+pull requests).
```

Comment:
I would highlight the advantages for the user in first place here: You get a local git checkout directory where you can work as usual inside and test your changes with local builds. Even without commit or pushing them.

It helps us on the server side with disk space as an addition, yes;)

Reply:
I would also mention that the content of the .obscpio archives appear as extracted source for the build process. So you can use any service processing these directories. Or you could also theoretically compile directly inside this trees, if your build tool (rpmbuild/...) supports it.

However, you want usually a tar ball as part of your src.rpm, therefore it gets created via the tar service.

Reply:
> I would highlight the advantages for the user in first place here: You get a local git checkout directory where you can work as usual inside and test your changes with local builds. Even without commit or pushing them.

Oh yeah, great point thanks :)

> I would also mention that the content of the .obscpio archives appear as extracted source for the build process.

I'm not sure I understand - how is that different to what you said in the previous comment about there being a local git checkout directory?  Are you saying that at build time there are *two* unpacked directories - one with a git checkout (including `.git` metadata) and another one which is the unpacked (obs)cpio?

> So you can use any service processing these directories.

Can you give an example of how that would be used?

> Or you could also theoretically compile directly inside this trees, if your build tool (rpmbuild/...) supports it.

Presumably that would make the .src.rpms useless though?  What would be the advantage?